src/GeneticSharp.Domain/Chromosomes/ChromosomeBase.cs

```diff
@@ -318,7 +318,7 @@ protected virtual void CreateGene(int index)
         /// </code>
         /// </remarks>
         /// </summary>        
-        protected virtual void CreateGenes()
+        public virtual void CreateGenes()
```

Comment:
See [my other comment](https://github.com/giacomelli/GeneticSharp/pull/88#discussion_r526074205). The new proposal is safer I suppose, and could potentially trigger only a minor version bump, but I think a Major version increase is still probably a good idea because of the new CreateNew + InitializeGenes protocol.